tests/integration/test_coloring_claudia.py

```diff
@@ -26,8 +46,8 @@ def test_count_isomorphism_bijection(self):
         self.assertEqual(1, count_isomorphism(g, h, coloring))
 
     def test_count_isomorphism_recursive(self):
-        g = graph_vertex4edge4()
-        h = graph_vertex4edge4()
+        g = self.v4e4
+        h = create_graph_helper([(1, 2), (2, 3), (2, 4), (3, 4)])
         G = preprocessing.remove_loners(g + h)
```

Comment:
Wel jammer dat dit dan nog zo moet en niet met de `self.v4e4` kan, of is daar wat op te verzinnen? Want op nieuwe regel 148 doe je dat wel